core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts

```diff
@@ -35,7 +35,7 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
  * This behavior does not vary across modes/directions.
  */
 configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
-  test.describe(title('picker-column-internal: disabled'), () => {
+  test.describe(title('picker-column-internal: disabled items'), () => {
```

Comment:
With changing the test name, you will need to manually delete the previous screenshots: 
- https://github.com/ionic-team/ionic-framework/blob/FW-5408/core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts-snapshots/picker-internal-disabled-ios-ltr-Mobile-Chrome-linux.png
- https://github.com/ionic-team/ionic-framework/blob/FW-5408/core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts-snapshots/picker-internal-disabled-ios-ltr-Mobile-Firefox-linux.png
- https://github.com/ionic-team/ionic-framework/blob/FW-5408/core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts-snapshots/picker-internal-disabled-ios-ltr-Mobile-Safari-linux.png
- https://github.com/ionic-team/ionic-framework/blob/FW-5408/core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts-snapshots/picker-internal-disabled-md-ltr-Mobile-Chrome-linux.png
- https://github.com/ionic-team/ionic-framework/blob/FW-5408/core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts-snapshots/picker-internal-disabled-md-ltr-Mobile-Firefox-linux.png
- https://github.com/ionic-team/ionic-framework/blob/FW-5408/core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts-snapshots/picker-internal-disabled-md-ltr-Mobile-Safari-linux.png

Reply:
🤦 good catch - I don't know why I thought the `test.describe` title change was a screenshot name change.